### ISSUE TITLE
#ut-fruit-heights

### ISSUE THREAD

**Issue by g14ntj** [ZRiddle/PokeAlertsOnboarding#61]
Channel Name: #ut-fruit-heights

Map Link: [Fruit Heights City](http://s2map.com/#order=latlng&mode=polygon&s2=false&points=41.051466,-111.913978%0A41.010096,-111.892518)

**Comment by g14ntj** [comment]
I'm happy to donate bots if it'll help get this one and/or Kaysville opened up.  Thanks!

**Comment by buckbeck** [comment]
I'd love that we have a crew of 6 or 7 down here that always have to go to north salt lake to play!

**Comment by caseyraymond** [comment]
Yeah please add this one sometime soon. See more rare Pokemon in this area than anywhere else at random, so with slack it'd be amazing
